src/scripts/components/embed/Embed.jsx

```diff
@@ -0,0 +1,59 @@
+import React, { Component } from "react";
+import { getDuration } from "../../services/audio-tools";
+import autoBind from "react-autobind";
+import Wavesurfer from "react-wavesurfer";
+import queryString from "query-string";
+
+class Embed extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      playing: false,
+      song: {}
```

Comment:
Since the audio files used here might not always be songs, maybe consider renaming this variable to something more general, like `audio` or `audioFile`

Reply:
Thanks Jay! I don't know why I always think about the audio as a song haha I think it's the musician inside of me. Nice catch, thank you!